targets/targets.json

```diff
@@ -334,11 +333,27 @@
             "SLEEP",
             "SPI",
             "SPISLAVE",
-            "STDIO_MESSAGES"
+            "STDIO_MESSAGES",
+            "USTICKER"
         ],
-        "default_lib": "small",
+        "overrides": {
+            "tickless-from-us-ticker": true,
+            "boot-stack-size": "0x300"
+        },
+        "supported_c_libs": {
+            "arm": [
+                "std", "small"
+            ],
+            "gcc_arm": [
+                "std", "small"
+            ],
+            "iar": [
+                "std"
+            ]
+        },
+        "c_lib": "small",
         "release_versions": [
```

Comment:
Please remove the `"release_versions"` altogether as it is no longer needed and avoids the future need to update the OS version.
Also please add an attribute `"supported_application_modes": ["bare-metal"]`.

Reply:
Thanks.  I pushed the commit for this.